spec/lib/telephony/lib/pinpoint/aws_credential_builder_spec.rb

```diff
@@ -0,0 +1,83 @@
+describe Telephony::Pinpoint::AwsCredentialBuilder do
```

Comment:
all these files that are `spec/lib/telephony/lib` .. let's remove the second `lib`